templates/osd-spec.yml.j2

```diff
@@ -0,0 +1,10 @@
+data_devices:
+{%if storage_node_disks is defined %}
+  paths:
+{% for disk in storage_node_disks %}
+    - '{{disk}}'
+{% endfor %}
+{% else %}
+  all: true
+{% endif %}
+osds_per_device: {{ osds_per_device }}
```

Comment:
From where this particular var/fact(osds_per_device) is coming from ?

Reply:
it is supposed to come from group vars , which i missed adding :)
let me get line back